guide/payments/overview.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: guide
+title: Transaction overview
+description: Principles to strive for while building bitcoin applications for end-users.
+nav_order: 4
+parent: Payments
+permalink: /guide/payments/overview/
```

Comment:
`/overview` doesn't give enough context imo, `/transaction-overview` or simply `transactions` looks better, especially since we have `guide/payments/` so it's confusing which page is `main`. I suggest `transactions` to make it consistent with other chapters.